rotkehlchen/validation/validation_schemas.py

```diff
@@ -0,0 +1,80 @@
+from typing import Any, Dict, Union
```

Comment:
Please move this to the `serialization/` parent directory. It should be there. And rename the file to schemas.py

So `serialization/schemas.py`